website/docs/docs/build/incremental-models.md

```diff
@@ -310,5 +310,61 @@ select ...
 
 </File>
 
+<VersionBlock firstVersiont="1.4">
+
+### About incremental_predicates
```

Comment:
i think it's meant to be an h3 but wanted to double check if this part of the `incremental_strategy` header or a separate thing?

```suggestion
### About incremental_predicates
```

Reply:
It's a separate piece, thanks for noticing that!